src/components/Button/index.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
```

Comment:
Is there any reason not to use `.jsx` as a file extension? I don't have a strong preference just wondering

Reply:
The reason is that rollup is setup with `js` files. I  tried briefly to move them into JSX but didn't manage (I don't know much about rollup) and didn't seem like a worthwhile investment of time. No strong preference either